### DISCUSSION TITLE
AttributeError

### DISCUSSION THREAD

**Discussion by ShangZhe-1999** [deepmodeling/deepmd-kit#457] [Q&A]
after executing  dp freeze command , the following error appears，AttributeError: 'NoneType' object has no attribute 'model_checkpoint_path'，How to solve this problem？

**Comment by amcadmus** [comment]
Could you please provide the version of the code and the training script?

**Reply by ShangZhe-1999** [reply]
My problem has been solved, and thank you for your reply

**Reply by amcadmus** [reply]
Could you provide the details on how the error happened and how you solved it? So anybody else encounters it can solve it quickly.

**Reply by ShangZhe-1999** [reply]
https://github.com/deepmodeling/deepmd-kit/tree/master/examples/water/data    https://github.com/deepmodeling/deepmd-kit/blob/master/examples/water/train/water_se_a.json    these are my jsonfile and data directory. In the begining, I only installed deepmd-kit, later, I installed DP-GEN, then executed dp train command, there was no AttributeError anymore.
